docs/client.md

```diff
@@ -0,0 +1,156 @@
+# Catalyst Client
+
+Catalyst allows you to retrieve info from BigCommerce's APIs through the Catalyst Client.<br /><br />
+
+```ts
+import { client } from '~/client';
+```
+
+The client simplifies what you need to do to handle requests and responses. For example, you don't need to construct request URLs, configure request headers, or parse JSON responses. The client uses the channel ID and authorization tokens configured in the `.env` file by default.
+
+## Methods
+
+- `fetch()`: allows you to interact with BigCommerce's [GraphQL Storefront API](https://developer.bigcommerce.com/docs/storefront/graphql). You can execute all queries and mutations available in [BigCommerce's Storefront graph](https://developer.bigcommerce.com/graphql-storefront/reference).
+
+- `fetchSitemapIndex(channelId?: string)`: fetches the sitemap index for the store, which provides a URL to the XML sitemap.
+
+- `getCanonicalUrl(channelId?: string)`: constructs and returns the canonical URL for the BigCommerce store. This URL is used for constructing various API endpoint URLs and is based on the store's hash and the provided or default channel ID.
+
+- `getGraphQLEndpoint(channelId?: string)`: returns the GraphQL endpoint URL for the store, incorporating the optional channel ID to specify which store channel's GraphQL API to access. This URL is used for sending GraphQL queries and mutations.
+
+- `requestLogger(document: string)`: provides a logging function to help debug and monitor performance. Logs the operation type, name, request duration, and query complexity.
```

Comment:
These are private methods and probably shouldn't be documented.

Reply:
Good point, just removed the 3 methods